.github/workflows/pull-request-checks.yml

```diff
@@ -21,9 +21,4 @@ jobs:
         if: github.event.pull_request.user.login != 'dependabot[bot]'
         with:
           checklist-items: |
-            New functionality includes testing.
-            All tests pass
-            New functionality has been documented.
-            New functionality has javadoc added
-            Commits are signed per the DCO using --signoff
-            Commit changes are listed out in CHANGELOG.md file (See: [Changelog](../blob/main/CONTRIBUTING.md#changelog))
+            Functionality includes testing.
```

Comment:
The "API changes" and "Public documentation" lines were not present in this verification script. I'm not sure if that was intentional (they are not always needed), so I didn't add them here.